src/App.css

```diff
@@ -12,16 +12,20 @@
     animation: App-logo-spin infinite 20s linear;
   }
 }
-
+footer {
```

Comment:
Great start! Go ahead and pull this styling and add it to it's own file called Footer.css